### ISSUE TITLE
Is there a way to generate a dynamic inventory with this collection

### ISSUE THREAD

**Issue by dsroark-dt** [ansible-collections/vmware.vmware_rest#186]
##### ISSUE TYPE
- Feature Idea

##### COMPONENT NAME
- I would assume this would be a new module that is built upon[ the generic inventory plugin](https://docs.ansible.com/ansible/latest/dev_guide/developing_inventory.html#inventory-plugins)

##### ADDITIONAL INFORMATION
Simply put this would have a plugin in the collection that allows us to do similar actions that can be done in the community.vmware dynamic inventory plugin without having to install VMWare's custom python libraries.

The inventory file would be called and match a pattern like `*vmware_rest.yml`, such as:

```yaml
ansible-playbook -i inv/my_vmware_rest.yml myplaybook.yml
```

If this is already possible then the request is for documentation in the main README

**Comment by Akasurde** [comment]
@dsroark-dt Thanks for reporting this feature.

**Comment by goneri** [comment]
We cannot build an inventory plugin with the original REST API because they are way too limited. However the new /rest/vcenter end-points of vSphere 7.0.2 seems to resolve the problem. @alinabuzachis is working on something similar.

tl;dr: we may build an inventory plugin in the future and it will only support vSphere 7.0.2 and above.

**Comment by ecchong** [comment]
Dynamic inventory plugin is still missing.  Is this feature being work on?

**Comment by mikemorency** [comment]
No inventory plugins will be added to this collection. However, the `vmware.vmware` collection has both VM and ESXI host inventory plugins
